src/iuse.cpp

```diff
@@ -2279,6 +2292,10 @@ int iuse::hammer( player *p, item *it, bool, const tripoint & )
 {
     // If anyone other than the player wants to use one of these,
     // they're going to need to figure out how to aim it.
+    if( p->is_mounted() ){
```

Comment:
Please keeps the comment where they are, they usually refer to the code directly below them. This comments speaks about aiming, it has clearly no connection with the mounted check. It's now only confusing the reader without adding information.

Same applies to the comments in the other iuse-functions.

Please follow the official code style: a space must be between `)` and `{`.

Reply:
Huh - yes, also it seems the astyle plugin didnt work correctly in VS, as I did run it. Ill try again.